tests/manager/sqlalchemy/test_session.py

```diff
@@ -0,0 +1,77 @@
+from unittest.mock import patch
+
+from palace.manager.core.config import Configuration
+from palace.manager.sqlalchemy.model.coverage import Timestamp
+from palace.manager.sqlalchemy.session import (
+    SessionManager,
+    json_serializer,
+    production_session,
+)
+from palace.manager.sqlalchemy.util import get_one
+from tests.fixtures.database import DatabaseTransactionFixture
+
+
+class TestSessionManager:
+    def test_initialize_data_does_not_reset_timestamp(
+        self, db: DatabaseTransactionFixture
+    ):
+        # initialize_data() has already been called, so the database is
+        # initialized and the 'site configuration changed' Timestamp has
+        # been set. Calling initialize_data() again won't change the
+        # date on the timestamp.
+        timestamp = get_one(
+            db.session,
+            Timestamp,
+            collection=None,
+            service=Configuration.SITE_CONFIGURATION_CHANGED,
+        )
+        assert timestamp is not None
+        old_timestamp = timestamp.finish
+        SessionManager.initialize_data(db.session)
+        assert old_timestamp == timestamp.finish
```

Comment:
This first test is just moved from tests/manager/sqlalchemy/test_util.py since this is a more appropriate location for it.